### PR TITLE
Use Daffodil SBT Plugin

### PR DESCRIPTION
**Pull request by stevedlawrence**

### CODE REVIEW

File: pairsTransform/build.sbt [hunk 1]
```diff
@@ -6,12 +6,4 @@ version := "0.1.0-SNAPSHOT"
...
-libraryDependencies ++= Seq(
-  "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.5.0" % "test",
-  "junit" % "junit" % "4.13.2" % "test",
-  "com.github.sbt" % "junit-interface" % "0.13.2" % "test"
-)
-
-testOptions += Tests.Argument(TestFrameworks.JUnit, "-v")
-
-crossPaths := false
+enablePlugins(DaffodilPlugin)
```
Comment (tuxji) [inline pairsTransform/build.sbt:9]: How come pairsTransforms' build.sbt doesn't need any library dependencies, but the others do?

**Review by tuxji — APPROVED** [review]
+1

File: pairsTransform/build.sbt (hunk 1 above)
Comment (stevedlawrence) [reply pairsTransform/build.sbt:9]: the plugin only automatically adds daffodil-tdml-processor, junit, and junit-iterface as test dependencies. This is usually all that is needed for schema projects.

But these other projects also use the Java/Scala API's so they need to manually add that as a dependency using the `daffodilVersion.value` to specify the version. We could add a new setting to the plugin (e.g. `daffodilUsesScalaAPI`, but it's rare enough that I'm not sure it's worth it.

**Review by mbeckerle — APPROVED** [review]
+1

**Pull request merged**